CHANGELOG.md

```diff
@@ -3,6 +3,7 @@ UNRELEASED
 
 * [ [#1227](https://github.com/digitalfabrik/integreat-cms/issues/1227) ] Correct URL and Path field in imprint API
 * [ [#1226](https://github.com/digitalfabrik/integreat-cms/issues/1226) ] Skip cache for childen and parents end points
```

Comment:
```suggestion
```
I'm not sure, should we remove entries from the changelog that do not make it into the release? Strictly speaking, it's not a change in comparison to the previous version.